HACKING.adoc

```diff
@@ -188,3 +189,92 @@ You might even be able to get some feedback early which could save you some time
 You can apply that label to a pull request that is complete and ready for review.
 
 Makes triaging easier.
+
+
+== Node package
+
+=== Initial setup
+
+First you must install and configure {uri-nodejs-download}[Node.js] on your machine.
+
+=== Setup a test project
+
+In order to test the Node package, you will need to create a test project:
```

Comment:
It should be clear where this folder should be created. It should be created adjacent to the clone of asciidoctor-reveal.js.